tools/cli/commands/docker.js

```diff
@@ -112,29 +115,79 @@ const setVolumes = argv => {
 		};
 	}
 
-	console.log( JSON.stringify( volumesObj ) );
-
 	writeFileSync( volumesBuiltFile, yaml.dump( volumesObj ) );
 };
 
+/**
+ * Checks whether the command should run in foreground
+ *
+ * @param {object} argv - argv
+ * @returns {boolean} whether command is running in foreground
+ */
+const isInForeground = argv => argv.detached || argv.ngrok;
```

Comment:
This seems backwards, at least the `argv.detached` bit. It runs in the foreground when `-d` is _not_ specified.

Reply:
resolved